Samples/Windowing/cpp_winui/SampleApp (Package)/Package.appxmanifest

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="7089e2db-be6f-4635-9abc-c1430d700d1a"
+    Publisher="CN=rokarman"
+    Version="1.0.0.0" />
+
+  <Properties>
+    <DisplayName>SampleApp (Package)</DisplayName>
+    <PublisherDisplayName>rokarman</PublisherDisplayName>
+    <Logo>Images\StoreLogo.png</Logo>
+  </Properties>
```

Comment:
Should probably not use my identity for the package. :) (goes for both projects)